frame/header.go

```diff
@@ -1,37 +1,34 @@
 package frame
 
-import (
-	"fmt"
-)
+// StreamID is a type alias for Short.
+type StreamID = Short
```

Comment:
All the types are currently declared in types.go, I would consider moving it there.